src/app/containers/Image/helpers/srcSetHelper.test.js

```diff
@@ -0,0 +1,47 @@
+import createSrcset from './srcSetHelper';
+
+describe('create srcset', () => {
+  const srcsetScenarios = [
+    {
+      originCode: 'test',
+      location: 'testland',
+      width: 1024,
+      expected:
+        'https://ichef.bbci.co.uk/news/240/test/testland 240w, https://ichef.bbci.co.uk/news/320/test/testland 320w, https://ichef.bbci.co.uk/news/480/test/testland 480w, https://ichef.bbci.co.uk/news/624/test/testland 624w, https://ichef.bbci.co.uk/news/800/test/testland 800w',
+      summary:
+        'should return a srcset with test in originCode and testland in location',
+    },
+    {
+      originCode: 'pips',
+      location: 'testland',
+      width: 1024,
+      expected: null,
+      summary: 'should return null with pips originCode',
+    },
+    {
+      originCode: 'test',
+      location: 'testland',
+      width: 640,
+      expected:
+        'https://ichef.bbci.co.uk/news/240/test/testland 240w, https://ichef.bbci.co.uk/news/320/test/testland 320w, https://ichef.bbci.co.uk/news/480/test/testland 480w, https://ichef.bbci.co.uk/news/624/test/testland 624w, https://ichef.bbci.co.uk/news/640/test/testland 640w',
+      summary: 'width of 640 should return srcset without 800w',
```

Comment:
```suggestion
      summary: 'width of 640 should return srcset with maximum allowed size of 640',
```

I'd change the summary to something like this, that captures the two things that happen: 

1. removes the sizes that exceed the original width
2. inserts a max size that is of the original width of the image